app/models/organization.js

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose';
+
+var Schema = mongoose.Schema;
```

Comment:
It would be cleaner to deconstruct on import this like `import {Schema} from 'mongoose';` instead of grabbing `Schema` afterwards here